PeerioTechnologies/peerio-mobile
# feat: auto capitalize file rename [branch ch4227]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/4227/mobile-capitalize-first-letter-of-file-folder-title-when-naming
#### Testing instructions  
- Upload a file to Peerio files and wait for name popup to show. Ensure that when the cursor is at the beginning of the textbox that the keyboard is in "Capital Letter" mode.
- Tap "Create a new Folder". Ensure that when the cursor is at the beginning of the textbox that the keyboard is in "Capital Letter" mode.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
